LibHR/Utils/test_utils.c

```diff
@@ -128,6 +127,10 @@ void copy_clover_force_cpu(suNf_field *out, suNf_field *in) {
     memcpy(out->ptr, in->ptr, 6 * out->type->gsize_gauge * sizeof(suNf));
 }
 
+void copy_staple_field_cpu(suNg_field *out, suNg_field *in) {
+    memcpy(out->ptr, in->ptr, 3 * out->type->gsize_gauge * sizeof(suNg));
+}
+
```

Comment:
This is for testing so it doesn't matter, but in general it's better not to use memcpy, memset, etc for performance reasons because of numa as these are single threaded.